sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/View.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.view;
+
+import com.google.auto.value.AutoValue;
+import io.opentelemetry.sdk.metrics.aggregator.AggregatorFactory;
+import javax.annotation.concurrent.Immutable;
+
+@AutoValue
+@Immutable
+public abstract class View {
+  public abstract AggregatorFactory getAggregatorFactory();
+
+  public static Builder builder() {
+    return new AutoValue_View.Builder();
+  }
+
+  @AutoValue.Builder
+  public abstract static class Builder {
```

Comment:
Hide the ctor for this so only View.builder() allows to create instances.

Reply:
done